docs/src/js/classes/Index.md

```diff
@@ -35,7 +35,7 @@
 
 #### Defined in
 
-[indices.ts:118](https://github.com/lancedb/lancedb/blob/9d178c7/nodejs/lancedb/indices.ts#L118)
+[indices.ts:118](https://github.com/universalmind303/lancedb/blob/833b375/nodejs/lancedb/indices.ts#L118)
```

Comment:
Having your fork in the URL doesn't seem desirable long term. Should this be fixed somehow?

Reply:
just removed the "Defined In" sections from the api reference docs